manuscript/01.introduction.md

```diff
@@ -11,8 +11,8 @@ Large-scale profiling studies can find molecular features that associate with dr
+Hidden Markov models (HMMs) provide an efficient strategy to infer discrete states from measurements when a series of co-dependent observations are made. An example of this is their wide-spread use in time series analysis, where each measurement is dependent on those that came before [@doi:10.1371/journal.pone.0223183]. Recognizing this co-dependence allows HMMs to make accurate inferences even in the presence of extremely noisy measurements since each neighboring measurement provides accumulating evidence for a prediction [@DOI:10.1126/scisignal.aao1917]. HMMs have been adapted to lineage trees (tHMMs) so that each measurement across the tree can similarly provide accumulating evidence for a prediction. Just like with time-series data, these models can provide very accurate predictions despite noisy measurements and limited information by recognizing the co-dependence between related measurements [@doi:10.1109/78.668544; @doi:10.1109/TSP.2004.832006]. tHMMs have been used in a multitude of applications, from image classification to comparative genomics [@pmid:18255546; @pmid:23762278]. In cells, these models have been fit to lineages collected from stem cells and bacteria colonies but have always required tailor-made implementations [@PMID:19628503; @doi:10.1101/488981]. Improvements in cell tracking and high-throughput imaging promise to make these models valuable techniques for studying the plasticity of heterogeneous cell populations. However, widespread use will require usable implementations that can be readily adapted to different experimental measurements, examples of unique insights revealed, and standards for experimental validation.
 
 <!-- Introduction to the paper -->
 
-Here, we develop a generic implementation of tHMMs with a defined interface for integrating diverse types of measurements on cell lineage trees. We use this to analyze how populations of breast cancer cells respond to therapy with a cell cycle reporter. Single cell measurements of the cell cycle revealed extensive variation in drug response that is not captured in population-level measurement. This model allows us to quantify the dynamics and phenotypic features of heterogeneity in drug response. We are able to predict the most likely number of phenotypically distinct subpopulations, the characteristics of those subpopulations, transition probabilities from one state to another, and each cell's expected state. This work, therefore, provides a flexible phenotype-driven route to discovering cell-to-cell variation in drug response, demonstrates an overall strategy for quantifying the dynamics of cell heterogeneity, and implements a very general software tool for widespread use of tHMM models.
```

Comment:
It's not just parent-child relationships, though...

Reply:
I agree but thought basically other relationships stem from that.. I meant we don't exactly focus on say sister-sister relationship. Should I explain more in that paragraph?

Reply:
This sentence is modified to "We use the relationship between the cells (that share the same ancestor or belong to the same lineage tree) to analyze how populations of breast cancer cells respond to therapy with a cell cycle reporter."